src/omegga/plugin.ts

```diff
@@ -280,22 +290,28 @@ export class PluginLoader {
   commands: Record<string, IPluginCommand & { _plugin: Plugin }>;
   documentation: Record<string, IPluginDocumentation & { _plugin: Plugin }>;
 
-  constructor(pluginsPath: string, omegga: Omegga) {
-    this.path = pluginsPath;
+  constructor(workDir: string, omegga?: Omegga) {
+    this.path = path.join(workDir, soft.PLUGIN_PATH);
     this.omegga = omegga;
     this.store = Datastore.create({
-      filename: path.join(omegga.dataPath, soft.PLUGIN_STORE),
+      filename: path.join(workDir, soft.DATA_PATH, soft.PLUGIN_STORE),
       autoload: true,
     });
     this.store.persistence.setAutocompactionInterval(1000 * 60 * 5);
     this.formats = [];
     this.plugins = [];
 
-    // soon to be deprecated !help
-    omegga.on('chatcmd:help', this.showHelp.bind(this));
+    Logger.verbose('Creating loading plugins');
```

Comment:
this was a typo in the original command - should just be `Loading plugin formats`